src/client/app/frameworks/i18n/effects/multilingual.effect.ts

```diff
@@ -6,6 +6,7 @@ import { Store, Action } from '@ngrx/store';
 import { Effect, Actions } from '@ngrx/effects';
 import { includes, map } from 'lodash';
 import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/map';
```

Comment:
Hi @NathanWalker, I want to thank you and all contributors too, this repository helped much developing unit tests required for my packages. It's a very useful reference for angular development and I'd love to keep contributing to this project with various enhancements and features.

If we talk about that import, I tried to work on the project using a few different IDE's. Actually, this import does not seem necessary except than on the stack VS2015 (Update 3) + Resharper 2016.2.2.

I'm also aware of TypeScript support is not perfect yet on Resharper, but I observed that importing rxjs/add/operator/map solves the IDE intellisense issue - until they improve the TS intellisense functionality on further releases.

So that import is not a fix, it's just a workaround. However, with some tree shaking, the compiled code will be optimized for production.

So, I leave it to you :)